plugins/fileservices/fileservices.cpp

```diff
@@ -3140,3 +3140,30 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+CriticalSection  noCommonDefCrit;
```

Comment:
minor: should be static

trivial: 2 spaces.

Reply:
Changed